auth-external-api/src/main/java/uk/gov/di/authentication/external/services/UserInfoService.java

```diff
@@ -102,6 +102,9 @@ private void addClaimsFromSession(AuthSessionItem authSession, UserInfo userInfo
         userInfo.setClaim(
                 AuthUserInfoClaims.VERIFIED_MFA_METHOD_TYPE.getValue(),
                 authSession.getVerifiedMfaMethodType());
+        userInfo.setClaim(
+                AuthUserInfoClaims.CURRENT_CREDENTIAL_STRENGTH.getValue(),
+                authSession.getCurrentCredentialStrength());
```

Comment:
Rather than always setting the claim, orch should always request it as outlined in this ticket
https://govukverify.atlassian.net/browse/ATO-1129

This is why you're getting the merge conflicts with main I expect - I've just merged the changes that implements this on auth's side.

Reply:
Feels a bit strange for this as from an orch perspective we dont care about the currentCredentialStrength . We just hold on to it. 

But if this is the pattern for things to be passed onto orch will implement.